### PR TITLE
win: clean up Windows related macros in runtime/flang

### PR DESCRIPTION
**Pull request by tdusnoki**

### CODE REVIEW

File: runtime/flang/descIntrins.c [hunk 1]
```diff
@@ -18,10 +18,7 @@
 #include <string.h>
 #include "fioMacros.h"
 /* macros for entries */
-#if defined(WINNT) && !defined(WIN64) && !defined(UXOBJS) && !defined(CROBJS)
```
Comment (kaadam) [inline runtime/flang/descIntrins.c:21]: I think you should rebase this PR once #1156 lands.

File: runtime/flang/entry.c [hunk 2]
```diff
@@ -14,16 +14,16 @@
 #include "stdioInterf.h"
 #include "fioMacros.h"
 
-#if defined(WIN64) || defined(WIN32)
```
Comment (kaadam) [inline runtime/flang/entry.c:17]: same as above.

File: runtime/flang/fioMacros.h [hunk 3]
```diff
@@ -25,20 +25,9 @@
 
 /* special argument pointers */
 
-#if defined(TARGET_WIN) || defined(WIN64) || defined(WIN32)
+#if defined(TARGET_WIN) || defined(_WIN64)
 WIN_IMP __INT_T ENTCOMN(0, 0)[4];
 WIN_IMP __STR_T ENTCOMN(0C, 0c)[1];
-#elif defined(WINNT)
```
Comment (kaadam) [inline runtime/flang/fioMacros.h:31]: same here

File: runtime/flang/initpar.c [hunk 4]
```diff
@@ -66,21 +66,18 @@ static struct {
 /* common blocks containing values for inlined number_of_processors()
    and my_processor() functions */
 
-#if defined(WIN64) || defined(WIN32)
```
Comment (kaadam) [inline runtime/flang/initpar.c:69]: see #1156

File: runtime/flang/initpar.c [hunk 5]
```diff
@@ -97,8 +94,8 @@ __fort_ncpus()
   return __fort_tcpus;
 }
 
-#if defined(WINNT)
-#if !defined(WIN64) && !defined(WIN32)
+#if defined(_WIN64)
+#if !defined(_WIN64)
```
Comment (kaadam) [inline runtime/flang/initpar.c:98]: It seems this condition a bit exclusive, this block could be removed.

File: runtime/flang/global.h [hunk 6]
```diff
@@ -364,7 +364,7 @@ extern int __fortio_check_format(void);
 extern int __fortio_eor_crlf(void);
 extern VOID __fortio_fmtinit(void);
 extern VOID __fortio_fmtend(void);
-#if defined(WINNT)
+#if defined(_WIN64)
```
Comment (kaadam) [inline runtime/flang/global.h:367]: Is it still necessary? Could EOR_CRLF be set via __fortio_eor_crlf?

**Review by bryanpkc — APPROVED** [review]

**Comment by tdusnoki** [comment]
@pawosm-arm @kiranchandramohan Would you mind taking a look? Thanks

**Review by pawosm-arm — APPROVED** [review]

**Comment by bryanpkc** [comment]
@tdusnoki Sorry for the delays. Could you rebase this branch?

**Pull request merged**